### DISCUSSION TITLE
Memory Bookmark Pause if

### DISCUSSION THREAD

**Discussion by LiquifiedSnow** [RetroAchievements/RAIntegration#785] [Ideas]
Suggesting adding a value component to pause in the memory bookmark section. Pause if value = 0a for example. I think it would help with trying to determine if you can use a specific address if nothing else but what you're looking for triggers that exact value instead of staring at it or having it pause anytime it changes

**Comment by Jamiras** [comment]
You can do this with a local achievement. Create a single condition "Value = 0a", and check the "Pause on Trigger" and "Active" checkboxes. When the value becomes 0a, the achievement will trigger and cause the emulator to pause.

**Comment by LiquifiedSnow** [comment]
Big Brain! Thanks for that idea.

**Comment by MagmarFire** [comment]
Seeing this fully realized as a bookmark feature would still be nice, though, I think. The achievement method feels like a clunky workaround more than anything, especially since it requires a dev to not only create a new achievement and clutter the editor but also reactivate the achievement each time they want to retest. UX for the current pause functionality is better because of how few clicks it requires.
